src/analyses/dependence_graph.cpp

```diff
@@ -196,7 +196,7 @@ void dep_graph_domaint::transform(
   ai_baset &ai,
   const namespacet &ns)
 {
-  dependence_grapht *dep_graph=dynamic_cast<dependence_grapht*>(&ai);
+  dependence_grapht *dep_graph=static_cast<dependence_grapht*>(&ai);
   assert(dep_graph!=nullptr);
```

Comment:
`static_cast` doesn't give null for a bad cast - no need to check for this (couple of instances of this).